sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.JsonSchema/api/Microsoft.Azure.Data.SchemaRegistry.JsonSchema.netstandard2.0.cs

```diff
@@ -0,0 +1,27 @@
+namespace Microsoft.Azure.Data.SchemaRegistry.JsonSchema
+{
+    public abstract partial class SchemaRegistryJsonSchemaGenerator
+    {
+        protected SchemaRegistryJsonSchemaGenerator() { }
+        public abstract string GenerateSchemaFromObject(System.Type dataType);
+        public virtual bool IsValidToSchema(object data, System.Type dataType, string schemaDefinition) { throw null; }
+    }
+    public partial class SchemaRegistryJsonSerializer
+    {
+        protected SchemaRegistryJsonSerializer() { }
+        public SchemaRegistryJsonSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client, Microsoft.Azure.Data.SchemaRegistry.JsonSchema.SchemaRegistryJsonSchemaGenerator jsonSchemaGenerator) { }
+        public SchemaRegistryJsonSerializer(Azure.Data.SchemaRegistry.SchemaRegistryClient client, string groupName, Microsoft.Azure.Data.SchemaRegistry.JsonSchema.SchemaRegistryJsonSchemaGenerator jsonSchemaGenerator) { }
+        public object Deserialize(Azure.Messaging.MessageContent content, System.Type dataType, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask<object> DeserializeAsync(Azure.Messaging.MessageContent content, System.Type dataType, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask<TData> DeserializeAsync<TData>(Azure.Messaging.MessageContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public TData Deserialize<TData>(Azure.Messaging.MessageContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public Azure.Messaging.MessageContent Serialize(object data, string schemaId, System.Type dataType = null, System.Type messageType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public Azure.Messaging.MessageContent Serialize(object data, System.Type dataType = null, System.Type messageType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask<Azure.Messaging.MessageContent> SerializeAsync(object data, string schemaId, System.Type dataType = null, System.Type messageType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I think that its worth considering having a deserialize overload that accepts a string that represents the schema.  If they have the schema, no reason to generate it.   

Thoughts?

Reply:
Assuming you meant serialize overload? But yes I think that's a good idea

Reply:
Yes, that's what I meant.  😄